scripts/start.sh

```diff
@@ -8,8 +8,8 @@ fi
 
 export $(cat .env | grep NODE_ENV | xargs)
```

Comment:
@saolsen / @izakp - this is probably a dumb question but is there a security issue with exporting these ENV vars to the logs in prod?

Reply:
The `.env` file shouldn't be present in prod / staging as all env vars are injected by Kubernetes - otherwise all docker container stderr output is forwarded via [syslog to Papertrail over TLS](https://github.com/artsy/substance/blob/master/services/logspout.yml#L36) so unless we are concerned about it being in Papertrail logs (data at rest is generally good to think about as an attack surface) so I wouldn't emit any sensitive info in logs in general... here it seems okay though.

Reply:
👍